cdap-docs/admin-manual/source/installation/integrations.txt

```diff
@@ -73,8 +73,7 @@ Prerequisites
 Download
 --------
 
-Download the CDAP CSD (Custom Service Descriptor): it can be found on the `Cask Resources
-webpage <http://cask.co/resources/#cdap-cds>`__.
+Download the CDAP CSD (Custom Service Descriptor): it can be found `here <http://repository.cask.co/downloads/co/cask/cdap/csd/2.7.0/CDAP-2.7.0-1.jar>`__.
```

Comment:
What using as text (instead of "it can be found here"):

```
Download the CDAP CSD (Custom Service Descriptor): `download JAR file <http://repository.cask.co/downloads/co/cask/cdap/csd/2.7.0/CDAP-2.7.0-1.jar>`__.
```